Дз5/form.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="css/style_for_form.css">
+  <title>Формы</title>
+</head>
+
+<body>
+  <h1>Форма</h1>
+  <hr>
+  <form action="/">
+    <div>
+      <p>Логин</p>
+      <input type="text" placeholder="логин">
+      <p>Пароль</p>
+      <input type="password" placeholder="пароль">
+    </div>
+    <div>
+      <p>Оцените наш сайт</p>
+      <label class="grade">
+        <input type="radio" name="gra">
+        <span>Отлично</span>
+      </label><br>
+      <label class="grade">
+        <input type="radio" name="gra">
+        <span>Хорошо</span>
+      </label><br>
+      <label class="grade">
+        <input type="radio" name="gra">
+        <span>Плохо</span>
+      </label>
+    </div>
+    <div>
+      <p>Как вы проводите свое свободное время?</p>
+      <label class="leisure">
+        <input type="checkbox" name="lei">
+        <span>Читаю книгу</span>
+      </label><br>
+      <label class="leisure">
+        <input type="checkbox" name="lei">
+        <span>Смотрю ТВ</span>
+      </label class="leisure"><br>
+      <label class="leisure">
+        <input type="checkbox" name="lei">
+        <span>Читаю книгу</span>
+      </label><br>
+      <label class="leisure">
+        <input type="checkbox" name="lei">
+        <span>Слушаю музыку</span>
+      </label><br>
+      <label class="leisure">
+        <input type="checkbox" name="lei">
+        <span>Гуляю с друзьями</span>
+      </label>
+    </div>
+    <div>
+      <p>Какие стили музыки вы слушаете?</p>
+      <select name="form_select" size="4" multiple="multiple">
+        <option value="Поп">Поп</option>
+        <option value="Рэп">Рэп</option>
+        <option value="Рок">Рок</option>
+        <option value="Джаз">Джаз</option>
+        <option value="Кантри">Кантри</option>
+        <option value="Блюз">Блюз</option>
+      </select>
+    </div>
+    <div class="number">
+      <p>Количество</p>
+      <button class="number-minus" type="button" onclick="this.nextElementSibling.stepDown()">-</button>
+      <input type="number" min="0" value="0" readonly>
+      <button class="number-plus" type="button" onclick="this.previousElementSibling.stepUp()">+</button>
```

Comment:
![image](https://user-images.githubusercontent.com/47972692/169521026-4cf396a3-d83f-4e15-a0c9-1ecb44433ade.png)
not safe for browser support